webconsole/webconsole_default.go

```diff
@@ -2,12 +2,23 @@
 
 package webconsole
 
+//go:generate go run github.com/rakyll/statik -f -src=./default -p=webconsole -dest=../ -tags=!webconsole
+
 import (
 	"github.com/codenotary/immudb/pkg/logger"
 	"net/http"
+	"github.com/rakyll/statik/fs"
 )
 
-func SetupWebconsole(_ *http.ServeMux, _ logger.Logger, _ string) error {
-	// no op
+func SetupWebconsole(mux *http.ServeMux, l logger.Logger, addr string) error {
+	statikFS, err := fs.New()
+	if err != nil {
+		return err
+	}
+	l.Infof("Webconsole not built-in")
+	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		http.Redirect(w, r, "/missing/", http.StatusMovedPermanently)
```

Comment:
Wouldn't it be better to use some temporary redirect here? Browsers could remember such permanent redirection (including rewriting bookmarks etc) that could be problematic if one wants to switch back to the built-in webconsole.

Reply:
You are right.